client/src/App.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import index from './index.jsx';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App/>.div);
+  console.log('true')
+  ReactDOM.unmountComponentAtNode(div)
+})
```

Comment:
Please install a linter immediately so that your tooling can tell you issues w/ missing semicolons. This way, your PR reviewers don't have to tell you themselves.